inotify/inotify_linux.go

```diff
@@ -134,6 +135,22 @@ func (w *Watcher) RemoveWatch(path string) error {
 // received events into Event objects and sends them via the Event channel
 func (w *Watcher) readEvents() {
 	var buf [syscall.SizeofInotifyEvent * 4096]byte
+	var eventArray []*Event
```

Comment:
`eventArray ` will be shared by goroutine, which seems more prone to problems.

Reply:
hi，it must be accesse with lock，you could see that below’

Reply:
It's just my confusion, not the code.
_Instead of communicating through shared memory, share memory through communication._